qutip/tests/cqobjevo_compiled_coeff_827871948660708379.pyx

```diff
@@ -0,0 +1,41 @@
+#!python
```

Comment:
These files should not be included, could you add it to gitignore.
Without cython,".py" temp files will be created instead, but the "compiled_coeff" part is constant.

Reply:
Done, thanks for pointing out.